ante/cosmos/setup_test.go

```diff
@@ -16,30 +14,29 @@ import (
 	"github.com/cosmos/cosmos-sdk/testutil/testdata"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
-
 	"github.com/ethereum/go-ethereum/common"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
-
-	"github.com/evmos/evmos/v19/x/evm/statedb"
-	evmtypes "github.com/evmos/evmos/v19/x/evm/types"
-	feemarkettypes "github.com/evmos/evmos/v19/x/feemarket/types"
 	"github.com/evmos/os/ante"
 	evmante "github.com/evmos/os/ante/evm"
 	"github.com/evmos/os/crypto/ethsecp256k1"
 	"github.com/evmos/os/encoding"
 	"github.com/evmos/os/ethereum/eip712"
-	app "github.com/evmos/os/example_chain"
+	example_app "github.com/evmos/os/example_chain"
```

Comment:
nit: don't use snake_case to be consistent with the naming convention
```suggestion
	exampleapp "github.com/evmos/os/example_chain"
```

Reply:
sounds good, will address that 👍